.github/workflows/ci.yml

```diff
@@ -49,4 +49,4 @@ jobs:
         run: uv sync --all-extras --dev
 
       - name: Check Types
-        run: uv run mypy --install-types --non-interactive .
+        run: uv run mypy --install-types --non-interactive --explicit-package-bases .
```

Comment:
what's the reason for this change?

Reply:
same as above, was fiddling with things to understand why a typecheck workflow step was failing.